doc/getting_started/getting_started.rst

```diff
@@ -18,18 +18,20 @@ Once you have ROS 2 installed, make sure you have the most up to date packages:
   sudo apt update
   sudo apt dist-upgrade
 
-Install `Colcon <https://docs.ros.org/en/foxy/Tutorials/Colcon-Tutorial.html#install-colcon>`_ the ROS 2 build system: ::
+Install `Colcon <https://docs.ros.org/en/foxy/Tutorials/Colcon-Tutorial.html#install-colcon>`_ the ROS 2 build system with `mixin <https://github.com/colcon/colcon-mixin-repository>`_: ::
```

Comment:
I use and like mixin, but I am not %100 sure we should recommend it as default. Maybe we can add a section like "hey there is a nice tool called mixin that most devs use."